### PR TITLE
[ETHEREUM-CONTRACTS] Small script & workflow fixes

### PR DESCRIPTION
**Pull request by d10r**

### CODE REVIEW

**Comment by d10r** [comment]
not gonna block this for https://github.com/superfluid-finance/protocol-monorepo/issues/1448

File: .github/workflows/handler.list-super-token.yml [hunk 1]
```diff
@@ -58,19 +61,16 @@ jobs:
             ops-scripts/resolver-list-super-token.js \
             : ${{ github.event.inputs.token_address }}
 
-        env:
```
Comment (hellwolf) [inline .github/workflows/handler.list-super-token.yml:61]:
```
.github/workflows/handler.run-ethereum-contracts-script.yml:          ETH_MAINNET_PROVIDER_URL: ${{ secrets.ETH_MAINNET_PROVIDER_URL }}
.github/workflows/handler.list-super-token.yml:          ETH_MAINNET_PROVIDER_URL: ${{ secrets.ETH_MAINNET_PROVIDER_URL }}
.github/workflows/handler.deploy-to-mainnet.yml:      ETH_MAINNET_PROVIDER_URL: ${{ secrets.ETH_MAINNET_PROVIDER_URL }}

```

other places reference to it.

Comment (hellwolf) [reply .github/workflows/handler.list-super-token.yml:61]: a cleanup PR can follow.

**Review by hellwolf — APPROVED** [review]
LGTM

**Pull request merged**